protocol/0055-TREA-on_chain_treasury.md

```diff
@@ -1,10 +1,10 @@
 # Network Treasury
 
-The Network Treasury is a set of accounts (up to 1 per asset supported by the network via the asset framework) that are funded by parties, deposits, or by direct transfers (e.g. a portion of fees, or from insurance pools at market closure).
+The Network Treasury is a set of accounts (up to 1 per asset supported by the network via the asset framework) that are funded by parties, deposits, by direct transfers (e.g. a portion of fees) or [governance transfers](./0028-GOVE-governance.md#5-transfers-initiated-by-governance).
```

Comment:
It may be worth being explicit what is the relation of the global insurance pool accounts and the rewards accounts to this network treasury.

Reply:
The accounts spec would probably be the place to be clear about the types, "level" (network/asset/reward scheme/market/party/etc.), ownership, control, and relationships between accounts. We could link there from here.

It might be worth getting one or two people with less detailed system knowledge to review the descriptions and make sure its all clear.

Reply:
I've added additional sentence here and more details in accounts spec. The details in accounts spec aren't exhaustive, but hopefully a good starting point to clarify things further in subsequent PR if need be.